src/web/handlers/FlingGestureHandler.ts

```diff
@@ -21,7 +21,7 @@ export default class FlingGestureHandler extends GestureHandler {
 
   private maxDurationMs = DEFAULT_MAX_DURATION_MS;
   private minVelocity = DEFAULT_MIN_VELOCITY;
-  private delayTimeout!: number;
+  private delayTimeout!: NodeJS.Timeout;
```

Comment:
```suggestion
  private delayTimeout!: ReturnType<typeof setTimeout>;
```

Reply:
Done in [3f244d2](https://github.com/software-mansion/react-native-gesture-handler/pull/2852/commits/3f244d27a397c68bf80abedc2c9ed1e450ebf534)